src/node/rpc/frontend.h

```diff
@@ -272,6 +272,21 @@ namespace ccf
       auto& metrics = endpoints.get_metrics(endpoint);
       metrics.calls++;
 
+      const auto signed_request = ctx->get_signed_request();
+      // On signed requests, the effective caller id is the key id that
+      // signed the request, the session-level identity is unimportant
+      // NOTE: this is only verified on line 324 (verify_client_signature)
```

Comment:
Nit: remove line number which is likely to change in the (near) future.

Reply:
Sure